packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -564,7 +565,8 @@ export class ProfilesUtils {
             await ProfilesUtils.readConfigFromDisk(true);
             ZoweLogger.info(vscode.l10n.t("Zowe profiles initialized successfully."));
         } catch (err) {
-            if (err instanceof imperative.ImperativeError) {
+            // JSON parsing errors in team config files will have suppressDump=true
+            if (err instanceof imperative.ImperativeError && !err.mDetails.suppressDump) {
```

Comment:
creative workaround for the `instanceof imperative.ImperativeError` check 😋 
I think this makes more sense than just showing the error for all imperative errors